packages/zoe/zoe.js

```diff
@@ -209,46 +252,7 @@ const makeZoe = (additionalEndowments = {}) => {
        * minted liquidity tokens to Zoe.
```

Comment:
What do you think about consolidating `escrowEmptyOffer` too? Is there a reason not to?

Reply:
Yeah, escrowEmptyOffer doesn't do any of the depositing of the payments, so there is little overlap apart from the creation of an offer record, and that is already one method call.

Reply:
On trying to use `depositPayment` in `escrowEmptyOffer` I remembered that `escrowEmptyOffer` is as it is now such that it is synchronous. If we switch to a `getZoeServices` approach, we should revisit this, possibly adding this method to the services